alert.css

```diff
@@ -0,0 +1,65 @@
+.alert {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.alert p {
+  margin-left: 0.7rem;
+}
+
+.success-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--alertgreen);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
+}
+.warning-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--alertyellow);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
+}
+.error-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--flatred);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
+}
+.feature-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--flatblue);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
+}
+.success-icon {
+  margin-top: 0.8rem;
+  margin-left: 0.8rem;
+  color: var(--flatgreen);
+}
+.warning-icon {
+  margin-top: 0.8rem;
+  margin-left: 0.8rem;
+  color: var(--fadedyellow);
+}
+.error-icon {
+  margin-top: 0.8rem;
+  margin-left: 0.8rem;
+  color: var(--alertred);
+}
+.feature-icon {
+  margin-top: 0.8rem;
+  margin-left: 0.8rem;
+  color: var(--alertblue);
+}
```

Comment:
Here also the margin style can be extracted instead of repetition.

Reply:
Done